ietf-optical-impairment-topology.tree

```diff
@@ -50,7 +52,8 @@ module: ietf-optical-impairment-topology
        |        |     +--ro roadm-noise-figure?
        |        |             l0-types:decimal-5-or-null
        |        +--:(roadm-drop-path)
-       |           +--ro roadm-drop-path* []
+       |           +--ro roadm-drop-path* [drop-path-id]
+       |              +--ro drop-path-id              uint16
        |              +--ro frequency-range
```

Comment:
Do we really need to define the path-id with three different name on the available choice. since the choice are mutually exclusive in my opinion we can just name the identifier as "path-id" in all the 3 available choice since the type of path is already identified in the container name. Not a strong opinion but it looks more simple to me.

Reply:
ok , we can change as general "path-id".